### PR TITLE
Add manifest version support for instance launch

### PR DESCRIPTION
**Pull request by DmZ**

### CODE REVIEW

File: qubell/cli/__main__.py [hunk 1]
```diff
@@ -1240,4 +1241,4 @@ def generate_session_token(refresh_token, verbose):
 
 
 if __name__ == '__main__':
-    entity(obj={})
\ No newline at end of file
+    entity(obj={})
```
Comment (vasichkin) [inline qubell/cli/__main__.py]: Why it is needed?

File: qubell/api/private/instance.py [hunk 2]
```diff
@@ -671,4 +673,4 @@ def not_found(hint=''):
         not_found('nothing within boundaries')
 
 
-activityLog = ActivityLog  # todo: remove this
\ No newline at end of file
+activityLog = ActivityLog  # todo: remove this
```
Comment (vasichkin) [inline qubell/api/private/instance.py]: Why??

File: qubell/cli/__main__.py (hunk 1 above)
Comment (DmZ) [inline qubell/cli/__main__.py]: This is an established practice to have newline at the end of file. POSIX standard defines "line" as  set of characters ending with "newline" character. UNIX tools will not work correctly if line is not terminated by newline. Also most of UNIX editors add newline to the last line automatically (this case). 
Reference to PEP8 checker: http://pep8.readthedocs.io/en/release-1.7.x/intro.html#error-codes (W292 - no newline at end of file)

Comment (vasichkin) [inline qubell/cli/__main__.py]: Right. I forgot how new line looks like.

**Pull request merged**